packages/core/test/lib/commands/config.js

```diff
@@ -27,68 +27,52 @@ describe("config", function() {
       }
     };
     config.network = "default";
-    config.logger = { log: val => val && memStream.write(val) };
+    config.logger = {log: val => val && memStream.write(val)};
   });
 
   beforeEach(() => {
     memStream = new MemoryStream();
-    memStream.on("data", function(data) {
+    memStream.on("data", data => {
       output += data.toString();
     });
   });
 
-  after("Cleanup tmp files", function(done) {
-    glob("tmp-*", (err, files) => {
-      if (err) done(err);
-      files.forEach(file => fs.removeSync(file));
-      done();
-    });
+  after("Cleanup tmp files", async () => {
```

Comment:
[Mocha says](https://mochajs.org/#arrow-functions) to use `function () {}` not `() => {}`:

```suggestion
  after("Cleanup tmp files", async function () {
```